example/frontend/app/util/KanaloaBridge.scala

```diff
@@ -3,15 +3,22 @@ package util
 import javax.inject.Inject
 
 import akka.actor.ActorSystem
+import akka.util.Timeout
 import api.ErrorResult
 import asobu.distributed.gateway.AbstractKanaloaBridge
 import kanaloa.reactive.dispatcher._
 import play.api.Configuration
+import KanaloaBridge._
+import concurrent.duration._
 
-class KanaloaBridge @Inject() (config: Configuration, system: ActorSystem) extends AbstractKanaloaBridge {
+class KanaloaBridge @Inject() (implicit configuration: Configuration, system: ActorSystem) extends AbstractKanaloaBridge {
```

Comment:
The `AbstractKanaloaBridge` requires those. So unless we change kanaloa again...